test/inductor/test_cudagraph_trees.py

```diff
@@ -1226,7 +1226,10 @@ def foo(x, y):
                     foo(*inps)
                 except Exception as e:
                     thrown = True
-                    self.assertTrue("at::cuda::blas::gemm<float>" in str(e))
+                    self.assertTrue(
```

Comment:
This unit test was failing for me locally, but is unrelated to this PR; just sneaking this fix in after discussing with @eellison